### PR TITLE
Inabox viewport reports size of body content

### PR DESCRIPTION
**Pull request by zombifier**
This is mostly useful for analytics purposes (e.g. making `${scrollHeight}` and `${scrollWidth}` get actual size of content, while `${elementHeight}` and `${elementWidth}` on root can report size of the content visible within the iframe.

### CODE REVIEW

**Comment by zombifier** [comment]
@jridgewell please take a look. I've been looking but I don't think there's anywhere in the code that will be affected by these functions not being no-ops anymore.

**Comment by jridgewell** [comment]
This goes to @lannka

File: src/inabox/inabox-viewport.js [hunk 1]
```diff
@@ -433,16 +433,30 @@ export class ViewportBindingInabox {
     }
   }
 
+  /** @override */
+  getScrollWidth() {
+    // Get actual width of document body, regardless of iframe size.
+    return this.getBodyElement()./*OK*/offsetWidth;
```
Comment (lannka) [inline src/inabox/inabox-viewport.js]: nitpick: `this.getScrollingElement().offsetWidth`

File: src/inabox/inabox-viewport.js [hunk 2]
```diff
@@ -433,16 +433,30 @@ export class ViewportBindingInabox {
...
+
+  /** @override */
+  getScrollHeight() {
+    // Get actual height of document body, regardless of iframe size.
+    return this.getBodyElement()./*OK*/offsetHeight;
+  }
+
+  /** @override */
+  getContentHeight() {
+    return this.getBodyElement()./*OK*/offsetHeight;
```
Comment (lannka) [inline src/inabox/inabox-viewport.js]: return this.getScrollHeight()

Comment (zombifier) [reply src/inabox/inabox-viewport.js]: Done.

File: src/inabox/inabox-viewport.js (hunk 1 above)
Comment (zombifier) [reply src/inabox/inabox-viewport.js]: Done.

**Comment by lannka** [comment]
tests kept failing. restarting again

**Comment by zombifier** [comment]
Merged head that should resolve the failing tests. Percy diff though, but that seems unrelated. Can you restart it again and merge it? Thanks.

**Comment by lannka** [comment]
finally :-)

**Pull request merged**